sequencer/src/api/endpoints.rs

```diff
@@ -135,10 +135,12 @@ where
                         })
                 }
             )?;
-            let common = &common.common().clone().context(CustomSnafu {
-                message: format!("failed to make proof for namespace {ns_id}"),
-                status: StatusCode::NOT_FOUND,
-            })?;
+            let VidCommon::V0(common) = &common.common().clone() else {
```

Comment:
I guess we will need to handle namespace proof verification at some point using V1 VID.